wikipedia-dark.user.css

```diff
@@ -1,7 +1,7 @@
 /* ==UserStyle==
 @name         Wikipedia Dark
 @namespace    StylishThemes
-@version      3.5.2
+@version      3.5.3
```

Comment:
Please don't version bump in a PR. That will happen as part of the release process.

Reply:
Allright, didn't know about that.